documentation/book/assembly-kafka-mirror-maker-replicas.adoc

```diff
@@ -0,0 +1,20 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-mirror-maker-replicas-{context}']
+
+= Replicas
+
+It is possible to run multiple Mirror Maker instances.
```

Comment:
To be clear by `instances` do you mean replicas in the `Deployment` sense? Because I initially read that as having multiple `KafkaMirrorMaker` resources, thus multiple `Deployments`, which works too, but you would need to ensure they're operating on a disjoint set of topics if their source and targets are the same, otherwise you'd get dupes.